contracts/contracts/rem.swap/src/rem.swap.cpp

```diff
@@ -0,0 +1,307 @@
+/**
+ *  @copyright defined in eos/LICENSE.txt
+ */
+
+#include <rem.swap/rem.swap.hpp>
+
+namespace eosio {
+
+    void swap::init( const name& user, const public_key& swap_key, const string& trx_id,
+                     const string& chain_id, const string eth_address, const asset& quantity,
+                     const uint32_t& swap_init_time ) {
```

Comment:
Why do you pass uint32_t and not block_timestamp?
With uint32 it will not be human readable.

Reply:
@Art17, done.